### PR TITLE
fix: use cosign attest instead of attach

### PR DESCRIPTION
**Pull request by thesayyn**

### CODE REVIEW

File: private/oci/sign_and_push.bzl [hunk 1]
```diff
@@ -8,9 +8,19 @@ tag="$(stamp "{TAG}")"
 
 [[ -n $EXPORT ]] && echo "$repository:$tag" >> $EXPORT
 
-"$(realpath {ATTACH_CMD})" --repository "$repository"
-"$(realpath {SIGN_CMD})" --repository "$repository" --key "$KEY" --attachment sbom
+# Push the image by its digest
+"$(realpath {PUSH_CMD})" --repository "$repository"
+
+# Attest the sbom
+"$(realpath {ATTEST_CMD})" --repository "$repository" --key "$KEY" --tlog-upload=false
```
Comment (loosebazooka) [inline private/oci/sign_and_push.bzl:15]: I think this is fine for now, but we probably want to upload everything to the tlog eventually.

**Review by loosebazooka — APPROVED** [review]

File: private/oci/sign_and_push.bzl (hunk 1 above)
Comment (thesayyn) [reply private/oci/sign_and_push.bzl:15]: yes. I didn't like the idea that bazel hitting a server other than the registry. if/when goes down, our cd pipeline will do down with it.

Comment (loosebazooka) [reply private/oci/sign_and_push.bzl:15]: keyless has to hit the tlog anyway for it to be useful :shrug:

Comment (thesayyn) [reply private/oci/sign_and_push.bzl:15]: hmm, I didn't think about that. not sure what should be done here. add tlog back?

Comment (thesayyn) [reply private/oci/sign_and_push.bzl:15]: sounds good

**Pull request merged**